assets/javascripts/discourse/initializers/patreon.js.es6

```diff
@@ -0,0 +1,56 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import TopicRoute from "discourse/routes/topic";
+
+let numTopicsOpened = 0;
+
+function initWithApi(api) {
+  const currentUser = api.getCurrentUser();
+
+  TopicRoute.reopen({
```

Comment:
Wonder if `TopicRoute.on("setupTopicController", function(event) {` wouldn’t be better, we try to avoid to reopen as much as possible.